File: sdkwork-app-sdk-csharp/Models/SpeakerStatisticsVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class SpeakerStatisticsVO
    {
        public int? TotalSpeakers { get; set; }
        public int? ActiveSpeakers { get; set; }
        public int? InactiveSpeakers { get; set; }
    }
}

File: sdkwork-app-sdk-java/src/main/java/com/sdkwork/app/model/SpeakerStatisticsVO.java
package com.sdkwork.app.model;

public class SpeakerStatisticsVO {
    private Integer totalSpeakers;
    private Integer activeSpeakers;
    private Integer inactiveSpeakers;

    public Integer getTotalSpeakers() {
        return this.totalSpeakers;
    }
    
    public void setTotalSpeakers(Integer totalSpeakers) {
        this.totalSpeakers = totalSpeakers;
    }

    public Integer getActiveSpeakers() {
        return this.activeSpeakers;
    }
    
    public void setActiveSpeakers(Integer activeSpeakers) {
        this.activeSpeakers = activeSpeakers;
    }

    public Integer getInactiveSpeakers() {
        return this.inactiveSpeakers;
    }
    
    public void setInactiveSpeakers(Integer inactiveSpeakers) {
        this.inactiveSpeakers = inactiveSpeakers;
    }
}
